docfx/api-override/Meadow.Foundation/Meadow.Foundation.ICs.IOExpanders.Mcp23xxx.md

```diff
@@ -0,0 +1,69 @@
+---
+uid: Meadow.Foundation.ICs.IOExpanders.Mcp23xxx
+remarks: *content
+---
+
+| Mcp23x08 | |
+|--------|--------|
+| Status | <img src="https://img.shields.io/badge/Working-brightgreen" style="width: auto; height: -webkit-fill-available;" alt="Status badge: working" /> |
+| Source code | [GitHub](https://github.com/WildernessLabs/Meadow.Foundation/tree/main/Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Mcp23x08) |
+| Datasheet(s) | [GitHub](https://github.com/WildernessLabs/Meadow.Foundation/tree/main/Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Mcp23x08/Datasheet) |
```

Comment:
Dont think its possible to have a datasheet link here since this is the base driver class